### PR TITLE
view a list, create a list and edit a list

### PR DESCRIPTION
**Pull request by lucieroland**

### CODE REVIEW

File: test/controllers/lists_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (glenntippett) [inline test/controllers/lists_controller_test.rb]: What does this Test file do, do we need it?

File: app/views/lists/index.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<h1> Your shopping Lists</h1>
+<h1> Your shopping Lists </h1>
```
Comment (glenntippett) [inline app/views/lists/index.html.erb]: h1 should be changed to h2
There should only be one h1 on the app, on the homepage, for SEO purposes

File: app/views/lists/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<%= form_for(@list) do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :due_date %>
+  <%= f.submit %>
+<% end %>
+
+export
```
Comment (glenntippett) [inline app/views/lists/_form.html.erb:7]: I don't think we need 'export' on this form?

File: test/controllers/lists_controller_test.rb (hunk 1 above)
Comment (lucieroland) [reply test/controllers/lists_controller_test.rb]: no idea.. I can't remember adding that, not even created that file

Comment (glenntippett) [reply test/controllers/lists_controller_test.rb]: haha ok, maybe double check if it's needed?

**Pull request merged**